### DISCUSSION TITLE
Bonus Labs

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#19] [Extra Credit]
### Due 7/26

This week's individual labs are extra credit and worth 15pts each:

[Bonus Lab #1](https://github.com/albertkun/211A-ASIAAM-191A/blob/master/Week_05/Lab/assignment.md)
[Bonus Lab #2](https://github.com/albertkun/211A-ASIAAM-191A/blob/master/Week_05/LabB/assignment.md)

## Submission
Post below the link to your bonus Lab, indicate if you are submitting `bonus lab #1` or `bonus lab #2`.

For example:
Bonus Lab #1: https://akochaphum.github.io/211A-191A-Lab-Assignments/week5b/
Bonus Lab #2: https://akochaphum.github.io/211A-191A-Lab-Assignments/week5c/

**Comment by abi-200** [comment]
Bonus Lab 2: https://abi-200.github.io/191a-labs/week5b/index.html

**Comment by abi-200** [comment]
Bonus Lab 1: https://abi-200.github.io/191a-labs/week5/

**Comment by cynthinee** [comment]
Bonus Lab 2: https://cynthinee.github.io/2021A-191A-Lab-Assignments/week5b/

**Comment by eandrusenki** [comment]
Bonus Lab 1:  https://eandrusenki.github.io/AsAmS-191A-Assignments/week5/bonuslab1/

**Comment by ramirezirvin** [comment]
Bonus Lab 1: https://ramirezirvin.github.io/191A-lab-assingments/week5/

**Comment by ramirezirvin** [comment]
Bonus Lab 2: https://ramirezirvin.github.io/191A-lab-assingments/week5b/
